tests/metrics/test_metrics.py

```diff
@@ -1553,3 +1557,247 @@ def test_equalised_odds_ratio(
             else:
                 eor = EqualisedOddsRatio(data, sensitive, real_target, predicted_target)
             eor.summary()
+
+
+expected_result_1 = [
+    {
+        "sensitive": ("Sex",),
+        "result": {
+            "HeartDisease": {("M",): 0.3726567804180811, ("F",): -0.3726567804180811}
+        },
+    }
+]
+
+
+expected_result_2 = [
+    {
+        "sensitive": ("Sex",),
+        "result": {
+            "HeartDisease": {("M",): 0.4219830636141608, ("F",): 2.369763353617309}
+        },
+    },
+    {
+        "sensitive": ("ChestPainType",),
+        "result": {
+            "HeartDisease": {
+                ("ASY",): 0.3917632113245289,
+                ("TA",): 0.9779803774692927,
+                ("NAP",): 1.3200622150699777,
+                ("ATA",): 3.612010526994567,
+            }
+        },
+    },
+    {
+        "sensitive": ("Sex", "ChestPainType"),
+        "result": {
+            "HeartDisease": {
+                ("M", "ASY"): 0.3468836645650188,
+                ("F", "ASY"): 0.5421518990141162,
+                ("M", "TA"): 0.6173483803022393,
+                ("M", "NAP"): 0.7690055427507021,
+                ("M", "ATA"): 2.1240334935639593,
+                ("F", "TA"): 3.869338673817374,
+                ("F", "ATA"): 4.671777837152278,
+                ("F", "NAP"): 5.173302314236593,
+            }
+        },
+    },
+]
+
+
+expected_result_3 = [
```

Comment:
add more details to the variable name. This is only for EqualisedOddsDifference

Reply:
but your variable expected_result_X is redefined throughout your test file

Reply:
Yes, for every test I have the expected result numbered corresponding to that test. case1, case2 ...

Reply:
I'm really not a fan of that, but I don't have arguments other than I don't find it really clear to what they relate. Do as you want then.

Reply:
It is the way I started the development and it rolled up to this but yeah I agree with you. I can see that it may cause an issue where the expected_value is not declared or is there was a typo in the name then another the expected _value for the previous test will be used. But it should be easy to fix. Do you think I should rename all the tests so that they are unique and relate to the test ?

Reply:
yep, I do think it is beter to change everything